apm-sniffer/apm-sdk-plugin/mongodb-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/mongodb/v4/support/MongoSpanHelper.java

```diff
@@ -43,6 +44,13 @@ public static void createExitSpan(String executeMethod, String remotePeer, Objec
         Tags.DB_TYPE.set(span, MongoConstants.DB_TYPE);
         SpanLayer.asDB(span);
 
+        if (operation instanceof EnhancedInstance) {
```

Comment:
What is the case not `instanceof`? Or this is just an extra check?

Reply:
For example,'CommitTransactionOperation' cannot obtain the corresponding database info